### ISSUE TITLE
Train cifar_resnet with SGDM(lr=0.1) leads to not learning

### ISSUE THREAD

**Issue by tmac1997** [keras-team/keras#12408]
I pulled latest example code and train cifar_resnet.py (ResNet56V1) with default parameters but just change optimizer from Adam to SGD with lr = 0.1 and momentum = 0.9. However, this simple modification leads to not learning just like below:

![53792421-9aedca80-3f66-11e9-99f5-22d220a02286](https://user-images.githubusercontent.com/11853126/53929307-45313380-40c8-11e9-9f1e-de299aacb728.png)

I've been monitoring the training process and after first several batches, the training loss suddenly explode. Then the total training ruined.

This is quite odd because many different papers claim that they can get better results using SGDM instead of Adam. And when training cifar10 with ResNet in Pytorch, SGDM works well. Could anyone help solve this problem? I've been implementing experiments in Keras and I don't know whether it's my problem or Keras's.

**Comment by tmac1997** [comment]
1.
